### PR TITLE
Remove SocketPoolContants

### PR DESCRIPTION
**Pull request by justmobilize**
Remove mis-spelled `SocketPoolContants` and place the constants where they should be

### CODE REVIEW

**Comment by justmobilize** [comment]
@dhalbert closed #204 and re-opened

File: adafruit_esp32spi/adafruit_esp32spi_socketpool.py [hunk 1]
```diff
@@ -60,12 +56,12 @@ def getaddrinfo(  # pylint: disable=too-many-arguments,unused-argument
         if not isinstance(port, int):
             raise ValueError("Port must be an integer")
         ipaddr = self._interface.get_host_by_name(host)
-        return [(SocketPoolContants.AF_INET, socktype, proto, "", (ipaddr, port))]
+        return [(SocketPool.AF_INET, socktype, proto, "", (ipaddr, port))]
```
Comment (dhalbert) [inline adafruit_esp32spi/adafruit_esp32spi_socketpool.py:59]: This should not be a qualified name now.
```suggestion
        return [(AF_INET, socktype, proto, "", (ipaddr, port))]
```

**Review by dhalbert — CHANGES_REQUESTED** [review]

File: adafruit_esp32spi/adafruit_esp32spi_socketpool.py (hunk 1 above)
Comment (justmobilize) [reply adafruit_esp32spi/adafruit_esp32spi_socketpool.py:59]: @dhalbert happy to add this and the pylint ignore, but thought this was cleaner (and it works):
image

Comment (dhalbert) [reply adafruit_esp32spi/adafruit_esp32spi_socketpool.py:59]: OK, I see, sorry. I was just testing by importing it into CPython, which did not complain. But it doesn't run.

**Review by dhalbert — APPROVED** [review]
This is cleaner, yes.

**Pull request merged**